src/controls/polyControl.ts

```diff
@@ -69,7 +69,7 @@ const polyActionHandler = (
 /**
  * Keep the polygon in the same position when we change its `width`/`height`/`top`/`left`.
  */
-const anchorWrapper = (
+export const factoryPolyActionHandler = (
```

Comment:
since this is an export now let's give it a name that is more understandable:
polygonAnchorPointWrapper? wrapWithPolygonAnchorPoint?

this is creating a new action handler rather than being a pure wrapper, but the name factory is a bit misleading because the action handler itself is passed as a parameter.

For me anchorPoint needs to be in the name

Reply:
It doesn't have to be exported